test/stripe/subscriptions/subscription_test.exs

```diff
@@ -35,6 +35,24 @@ defmodule Stripe.SubscriptionTest do
   end
 
   describe "delete/2" do
+    test "deletes a subscription when second argument is a list" do
+      assert {:ok, %Stripe.Subscription{} = subscription} =
+               Stripe.Subscription.delete("sub_123", [])
+
+      assert_stripe_requested(:delete, "/v1/subscriptions/#{subscription.id}")
+    end
+
+    test "deletes a subscription when second argument is a map" do
+      params = %{at_period_end: true}
+
+      assert {:ok, %Stripe.Subscription{} = subscription} =
+               Stripe.Subscription.delete("sub_123", params)
```

Comment:
inline params if you wanted while you are in here...

Reply:
Sure!